apps/src/templates/CheckboxDropdown.jsx

```diff
@@ -55,19 +56,13 @@ const CheckboxDropdown = ({
         <ul className={style.dropdownCheckboxUL}>
           {Object.keys(allOptions).map(optionKey => (
             <li key={optionKey} className="checkbox form-group">
-              <label>
-                <input
-                  type="checkbox"
-                  name={optionKey}
-                  value={optionKey}
-                  checked={checkedOptions.includes(optionKey)}
-                  onChange={onChange}
-                />
-                <i className="fa fa-solid" />
-                <Typography semanticTag="span" visualAppearance="body-one">
-                  {allOptions[optionKey]}
-                </Typography>
-              </label>
+              <Checkbox
+                checked={checkedOptions.includes(optionKey)}
+                onChange={onChange}
+                name={optionKey}
+                value={optionKey}
+                label={allOptions[optionKey]}
+              />
```

Comment:
Thank you for simplifying this! This looks so much better and is much more readable